From 23b4f2b567a118a11e56c41cd4f8cb863e1c2c06 Mon Sep 17 00:00:00 2001
From: Russell King <rmk+kernel@arm.linux.org.uk>
Date: Tue, 23 Jun 2015 11:51:43 +0200
Subject: [PATCH 050/139] staging: etnaviv: use definitions for constants

Use the etnaviv definitions for feature constants, rather than BIT()s.

Signed-off-by: Russell King <rmk+kernel@arm.linux.org.uk>
---
 drivers/staging/etnaviv/etnaviv_gpu.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/etnaviv/etnaviv_gpu.c b/drivers/staging/etnaviv/etnaviv_gpu.c
index 58b86d2..26a635b 100644
--- a/drivers/staging/etnaviv/etnaviv_gpu.c
+++ b/drivers/staging/etnaviv/etnaviv_gpu.c
@@ -254,7 +254,7 @@ static void etnaviv_hw_identify(struct etnaviv_gpu *gpu)
 
 	/* Disable fast clear on GC700. */
 	if (gpu->identity.model == 0x700)
-		gpu->identity.features &= ~BIT(0);
+		gpu->identity.features &= ~chipFeatures_FAST_CLEAR;
 
 	if ((gpu->identity.model == 0x500 && gpu->identity.revision < 2) ||
 	    (gpu->identity.model == 0x300 && gpu->identity.revision < 0x2000)) {
@@ -271,7 +271,8 @@ static void etnaviv_hw_identify(struct etnaviv_gpu *gpu)
 		gpu->identity.minor_features0 =
 				gpu_read(gpu, VIVS_HI_CHIP_MINOR_FEATURE_0);
 
-	if (gpu->identity.minor_features0 & BIT(21)) {
+	if (gpu->identity.minor_features0 &
+	    chipMinorFeatures0_MORE_MINOR_FEATURES) {
 		gpu->identity.minor_features1 =
 				gpu_read(gpu, VIVS_HI_CHIP_MINOR_FEATURE_1);
 		gpu->identity.minor_features2 =
-- 
2.5.1

